### DISCUSSION TITLE
module.generator 和 module.parse

### DISCUSSION THREAD

**Discussion by jianxingyao** [webpack/webpack#18870] [General]
对于 module.generator 和 module.parse  第一个选项和 Rule.generator  类型，但是第二个选项 module.parse 没理解什么意思，官方给的示例看起来和  module.generato 类似，希望有人能解答

For module.generator and module.parse the first option and the Rule.generator type, but the second option module.parse doesn't understand what it means, the official example looks similar to module.generato, I hope someone can answer

**Comment by alexander-akait** [comment]
This allows you to configure how to parse and how to generate files, for example you can use generator to convert images to DataURI
